rmf_traffic/src/rmf_traffic/agv/Graph.cpp

```diff
@@ -656,8 +686,7 @@ class Graph::Lane::Implementation
 
   Node exit;
 
-  bool has_door;
```

Comment:
Note that these fields were never being used anywhere, so I've decided to remove them. They can be added back in at any time without harming API or ABI.